llvm/lib/IR/Verifier.cpp

```diff
@@ -760,7 +761,7 @@ void Verifier::visitGlobalValue(const GlobalValue &GV) {
     if (const MDNode *AbsoluteSymbol =
             GO->getMetadata(LLVMContext::MD_absolute_symbol)) {
       verifyRangeMetadata(*GO, AbsoluteSymbol, DL.getIntPtrType(GO->getType()),
-                          true);
+                          true, false);
```

Comment:
```suggestion
                          /*IsAbsoluteSymbol=*/true, /*IsAddrSpaceRange=*/false);
```
Even though `IsAbsoluteSymbol` is not related to this PR.

Reply:
I'm actually thinking maybe it's time to introduce an enum to tell what the kind of range metadata it is.